From: Raphael Hertzog <rhertzog@hrnet.fr>
Last-Update: 1999-07-24
Description: Removed shell tricks
 The packaged stopped working in 1999 due to this. This patches
 substitutes the codes which used a shell to open perl to a hashbang
 calling perl interpreter directly.

diff --git a/html2ps b/html2ps
index c941b2c..1404164 100755
--- a/html2ps
+++ b/html2ps
@@ -1,6 +1,4 @@
-: # Use perl
-eval 'exec perl -S $0 "$@"'
-  if $running_under_some_shell;
+#! /usr/bin/perl
 
 # This is html2ps version 1.0 beta7, an HTML-to-PostScript converter.
 #   Copyright (C) 1995-2010 Jan Karrman.
